victoria_bank.rb

```diff
@@ -0,0 +1,151 @@
+require 'watir'
+require 'nokogiri'
+require 'pry'
+require 'json'
+require 'rubocop'
+require 'open-uri'
+require_relative 'transactions.rb'
+require_relative 'accounts.rb'
+
+class VictoriaBank  
+  def get_user_credentials
+    puts "Enter your username"
+    @username = gets.chomp
+    puts "Please enter your password"
+    @password = gets.chomp
+  end
+
+  def set_user_credentials
+    @browser.text_field(class: 'username').set(@username)
+    @browser.text_field(id: 'password').set(@password)
+    @browser.button(class: 'wb-button').click
+    validate_login
+  end
+
+  def validate_login
+    Watir::Wait.until do
+      if @browser.div(class: "error", text: "Invalid username or password").present?
+        puts "Inlavid username or password"
+        break
+      end
+      @browser.div(class: "contracts").present?
+    end
+  end
+
+  def init_session
+    @browser = Watir::Browser.new
+    @browser.goto "https://web.vb24.md/wb/#login"
+    Watir::Wait.until{ @browser.input(class: 'username').present? }
+  end
+
+  def run
+    init_session
+    get_user_credentials
+    set_user_credentials
+    access_accounts
```

Comment:
You should not access accounts if you got `Invalid username or password` error.

Reply:
I have a method that verifies the credentials.